### PR TITLE
Adding the user management UI

### PR DESCRIPTION
**Pull request by salmanmanekia**
Oauth feature

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="IoT platform admin"
-    />
-    <script src="/inject.js"></script>
+    <meta name="description" content="IoT platform admin" />
+    <!--REMOVING THE INJECTION OF SCRIPT WHEN RUNNING LOCALLY-->
+    <script type="text/javascript" src="/inject.js"></script>
```
Comment (ttsalonen) [inline public/index.html:10]: I don't understand the meaning of the above comment? Does it mean that the file does not exist when not running in docker container?

File: src/containers/UserManagement/UserManagement.jsx [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+import './UserManagement.css';
+
+import env from '../../config';
+const UserManagementContainer = styled.section`
+  margin-left: 18em;
+  background-color: #ffffff;
+`;
+
+const envVar = env();
+const baseDbApiUrl = envVar.BASE_DB_API_URL;
```
Comment (ttsalonen) [inline src/containers/UserManagement/UserManagement.jsx:17]: Env-var should be BASE_API_URL like in other components

File: public/index.html (hunk 1 above)
Comment (salmanmanekia) [reply public/index.html:10]: while running the project locally, there is a script not found problem in the browser console as inject.js is only generated with nginx while application is running in the production.
something to look into in the future.

Comment (ttsalonen) [reply public/index.html:10]: ok

**Pull request merged**